### ISSUE TITLE
Update docs for running tests

### ISSUE THREAD

**Issue by dangirsh** [HomerReid/scuff-em#165]
The docs currently reference `scuff-test-harness`, which doesn`t seem to exist anymore. There is also an `test-driver` script generated during the build, which I'm not sure how to use. Is there any automated way to run all tests?

**Comment by HomerReid** [comment]
No, because there are no tests :( The absence of a unit-test suite is the main obstacle preventing an official release. Helping to rectify this situation would be an extremely wonderful way for any interested users to contribute to the code while learning how to use it at the same time. I am working on a test suite, but it's not my main priority at the moment.

**Comment by HomerReid** [comment]
Feel free to keep this discussion going if you have further questions or contributions, or close the issue if it has been resolved to your satisfaction. Note that the need for a unit-test suite is already addressed by open issues #22 and #15.
